IPLD-Path.md

```diff
@@ -0,0 +1,20 @@
+# [WIP] IPLD Path v1
+
+An IPLD Path is a string identifier used for deep references into IPLD
+graphs.
+
+IPLD Path's are constructed following the same constraints as [URI Paths](https://tools.ietf.org/html/rfc3986#section-3.3).
+
+Similarly, the string `?` is reserved for future use as a query separator.
+
+# Path Resolution
```

Comment:
(Sorry, :bike: :building_construction: warning) Fwiw, I would refer to this as "traversal" rather than resolution.

And would rather refer to link traversal as loading another IPLD object (or node), rather than use the "block" terminology.  I think the "block" terminology is super confusing when you try to think about IPLD outside of IPFS; "block" terms come from IPFS's chunking, and IPLD should make just as much sense when applied to some plain file or HTTP stream or etc.

Reply:
Fair enough re: some other terms are already too overloaded to use.

I'm still dubious about "block" though.  You don't stack them on top of each other; they're not very blocklike :)  "Block" comes from taking a large stream of bytes and cutting it apart into *blocks*... and that term simply wouldn't have evolved if we had designed IPLD in a vacuum without first having spent a lot of time thinking the the practicalities of trying to break apart large binary files.

/2c

Reply:
@warpfork  I see block coming from block chain. You link up things and chain them together.

Though I agree and I'd be happy to have a better name. Though we should have this discussion on a separate issue. .o0(IPLD Atom, the smallest unit IPLD works with).